dnstable_convert.c

```diff
@@ -1058,12 +1060,19 @@ main(int argc, char **argv)
 
 		switch(c) {
 		case 'b':
-			dns_block_size = strtol(optarg, &end, 10);
-			if (*end != '\0' || dns_block_size < 1) {
-				fprintf(stderr, "Invalid DNS block size '%s'\n", optarg);
+			sscanf(optarg, "%d,%d", &dns_block_size, &dnssec_block_size);
```

Comment:
Using `sscanf` here is nice and succinct, but the `scanf` family of functions have a few complexities to be aware of (I tend towards "wary" of, in fact).

First, the return value of `sscanf` should be checked -- if you pass this a non-numeric string, `sscanf` will return 0, which should be treated as an error. 

Second, even checking the value of `sscanf` will miss `-b 65536,blah` -- `sscanf` will collect the 65536, return 1, and ignore the `,blah`.  You could either:
1. separately check the string for a `,` and require `sscanf` to return 2 if a `,` is present, or
2. do a second `strtol` call if the first ends with `*end == ','`. This also simplifies error reporting of invalid dns vs dnssec block size

Finally, the dnssec block size is an unsigned `size_t` (and the variables should be, too) -- if you stick with `sscanf`, use `%zu` for the format; if using `strtoXX` functions, use `strtoul`.